gprofiler/utils/perf.py

```diff
@@ -35,6 +38,72 @@
 )
 
 
+class SUPPORTED_PERF_EVENTS(Enum):  # pylint: disable=C0103
```

Comment:
nitpick - name it properly and remove the pylint. This is not C/C++. `class SupportedPerfEvent` is okay